org_status/status_providers/travis.py

```diff
@@ -6,6 +7,8 @@
 class TravisBuildStatus(StatusProvider):
     BadgeTemplate = ('https://api.travis-ci.org/{group}'
                      '/{repo}.svg?branch={branch}')
+    TRAVIS_STATUS_URL = ('https://pnpcptp8xh9k.statuspage'
+                         '.io/api/v2/status.json')
```

Comment:
Please let the TLD be with the domain name in the above line i.e. it should look like
```
('https://pnpcptp8xh9k.statuspage.io/'
 'api/v2/status.json')
```